netweaver/install_db.sls

```diff
@@ -92,4 +92,17 @@ remove_db_inifile_{{ instance_name }}:
     - require:
       - create_db_inifile_{{ instance_name }}
 
+{%- if netweaver.hana.ha_enabled|default(false) is sameas true %}
```

Comment:
I don't like the `ha_enabled` parameter.
I would rewrite it to `sr_enabled`

Reply:
done